apps/services/user-notification/src/app/modules/notifications/notifications.service.ts

```diff
@@ -32,30 +48,29 @@ export class NotificationsService {
     return await this.cacheManager.get(key)
   }
 
-  async mapLocale(locale: string | null | undefined): Promise<string> {
-    if (locale === 'en') {
-      return 'en'
-    }
-    return 'is-IS'
+  async mapLocale(locale?: string | null): Promise<string> {
+    return locale === 'en' ? locale : 'is-IS'
   }
+
   async getTemplates(
     locale?: string | null | undefined,
   ): Promise<HnippTemplate[]> {
-    locale = await this.mapLocale(locale)
+    const mappedLocale = await this.mapLocale(locale)
```

Comment:
Why renaming instead of just overwriting as it was?

Reply:
This is purely matter of style. I like not to override input parameters especially if they are to be mutated. It can lead to wrong code. In this case it does not matter but more of a principle with me